source/crud/tests/server_3_4/read/aggregate.yml

```diff
@@ -0,0 +1,17 @@
+data:
+    - {_id: 1, x: 'ping'}
+serverVersion: '3.4'
+
+tests:
+    -
+        description: "Aggregate with collation"
+        operation:
+            name: aggregate
+            arguments:
+                pipeline:
+                    - $match:
+                        x: 'PING'
+                collation: { locale: 'en_US', strength: 2 }
```

Comment:
Cross-referencing with https://docs.mongodb.com/master/reference/collation/#collation-document in case anyone else would like an explanation of `{strength: 2}` allowing for case-insensitive matching.